modules/engine/src/animation/timeline.d.ts

```diff
@@ -0,0 +1,39 @@
+interface ChannelProps {
```

Comment:
Nit: use `type` instead of `interface` if we are just describing the permitted fields of an object?